pkg/daemon/update.go

```diff
@@ -1084,6 +1084,7 @@ func getSupportedExtensions() map[string][]string {
 		"kerberos":             {"krb5-workstation", "libkadm5"},
 		"kernel-devel":         {"kernel-devel", "kernel-headers"},
 		"sandboxed-containers": {"kata-containers"},
+		"kernel-64k":           {"kernel-64k"},
```

Comment:
This does not look like the place it should be added. This is replacing the kernel so it should use the same logic as the kernel-rt extension.